pages/blog/[uid].tsx

```diff
@@ -88,6 +81,48 @@ export async function getStaticProps({
         }
       );
 
+    const linkedcategory =
```

Comment:
this should happen in the query above, there is no need to do a separate query here